src/buckets/actions/thunks.ts

```diff
@@ -92,7 +92,10 @@ export const getBuckets = () => async (
   }
 }
 
-export const createBucket = (bucket: OwnBucket) => async (
+export const createBucket = (
+  bucket: OwnBucket,
+  callback?: (bucket: Bucket) => void
```

Comment:
create bucket does one thing really well.  let's not add an optional callback to get the data from this.

Reply:
bucket names are unique and you know the name of the bucket since you passed it in.  why not retrieve the bucket that way?